main.tf

```diff
@@ -253,12 +279,17 @@ resource "aws_lb" "external_elb" {
   load_balancer_type = "application"
   security_groups    = [aws_security_group.web_sg.id]
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure one HTTPS listener is configured for AWS Load balancer through AWS LB Listener**
 
**Vulnerability**
Not configuring AWS Load balancer to have one HTTPS listener could impact the confidentiality of data in transit.
 
**Remediation**
Elastic Load Balancing uses Secure Sockets Layer (SSL) negotiation configurations, known as security policies, to negotiate connections between the clients and the load balancer. When you use HTTPS/SSL for your front-end connections, you can use either a predefined security policy or a custom security policy. You must deploy an SSL certificate on your load balancer.

source: tenablecs-